### ISSUE TITLE
New-NavContainer is slow on Windows 10

### ISSUE THREAD

**Issue by ajkauffmann** [Microsoft/navcontainerhelper#276]
New-NavContainer runs the command (Get-ComputerInfo).OsProductType to determine if the script is running on a server or a workstation.

On Window 10, the command Get-ComputerInfo is very slow. This is due to a bug on PowerShell: https://github.com/PowerShell/PowerShell/issues/3080

A temporary solution is to enable PowerShell remoting on Windows 10. This can be done by running Enable-PSRemoting -Force.
More info about this: http://www.happysysadm.com/2017/01/the-new-way-to-check-computer.html (down under update Feb 9th, 2017)

There is an alternative and even quicker way to check if the computer is a workstation or a server:
```
$osInfo = Get-CimInstance -ClassName Win32_OperatingSystem
$osInfo.ProductType
```

Returns:
1 = Workstation
2 = Domain controller
3 = Server

(Source: https://stackoverflow.com/questions/48592120/how-do-i-find-out-from-powershell-if-i-am-on-a-server-or-workstation)

**Comment by freddydk** [comment]
Seriously - I am working on this right now:-)
Also because I use the same inside the container - and that fails in some cases (bug from yesterday)

**Comment by freddydk** [comment]
Just checked in the fix in the dev branch - have a few other things to check before publishing

**Comment by ajkauffmann** [comment]
Thanks, will test them when ready!

Tip: you might want to apply the same change to start.ps1 in nav-docker. The change to replace Get-ComputerInfo I mean. ;-)

**Comment by freddydk** [comment]
Fixed in 0.4.2.6
